src/rules-engine/Property.ts

```diff
@@ -6,22 +6,43 @@
 
 'use strict';
 
-const assert = require('assert');
-const AddonManager = require('../addon-manager');
-const Constants = require('../constants');
-const Things = require('../models/things');
-const {EventEmitter} = require('events');
-const Events = require('./Events');
+import assert from 'assert';
+import AddonManager from '../addon-manager';
```

Comment:
If you switch AddonManager, Thing, and Things back to `require()` for now, the tests will work better.